lib/ui/instrumentpanel.js

```diff
@@ -383,27 +385,22 @@ InstrumentPanel.prototype.deserializeWidget = function(pageID, oneWidgetData, i)
   return widget;
 }
 
-
-InstrumentPanel.prototype.onLayoutChange = function(layout) {
-  this.getCurrentPage().layout = layout;
-  this.persist();
-}
-
 InstrumentPanel.prototype.onWidgetChange = function(widget) {
   this.pushGridChanges();
 }
 
-
 InstrumentPanel.prototype.persist = function() {
+  console.log('start timer persist');
   if (this.currentPage !== notificationPageId) {
     var that = this;
     if (this.handleSaveLayoutDelay !== null) {
       clearTimeout(this.handleSaveLayoutDelay);
     }
     this.handleSaveLayoutDelay = setTimeout(() => {
       that.handleSaveLayoutDelay = null;
+      console.log('persist => storeGrid');
```

Comment:
Do we want to keep these? We could add `debug` if we want optional logging, it works also in the browser.

Reply:
This is not essential, it was to trace the auto-save function calls of the layout.
In a future PR, I will add a menu for the layout to: save / export / import.